seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -2703,12 +2680,12 @@
       "title": "Directed and Undirected Graphs",
```

Comment:
Agreed. I can change it to something like what we did for regular expressions http://beta.freecodecamp.com/en/challenges/regular-expressions/introduction-to-the-regular-expression-challenges.